src/routes/notifications/notifications.controller.spec.ts

```diff
@@ -68,23 +68,39 @@ describe('Notifications Controller (Unit)', () => {
         .with('safeRegistration', safeRegistrations)
         .build();
       mockNetworkService.get.mockImplementation((url) => {
-        const getChainUrlPattern = `${safeConfigUrl}/api/v1/chains/`;
-        if (url.includes(getChainUrlPattern)) {
+        if (
+          chains.some(
+            (chain) =>
+              url === `${safeConfigUrl}/api/v1/chains/${chain.chainId}`,
+          )
+        ) {
           return Promise.resolve({ data: sample(chains) });
```

Comment:
I think using `sample` can potentially cause issues right? We might not match the chain returned. 

You have a different implementation using `find` in this PR which I think is better wdyt?

Reply:
See my comment [here](https://github.com/5afe/safe-client-gateway-nest/pull/267#discussion_r1098651611) instead